### DISCUSSION TITLE
Support for folders

### DISCUSSION THREAD

**Discussion by franmc01** [jonathanlermitage/intellij-extra-icons-plugin#92] [Ideas]
The idea is to make the plugin more user friendly. At the moment it would take time to create the styles for the folders but you could start adding the most common according to the technologies

![image](https://user-images.githubusercontent.com/50973629/163516601-5ef0cf91-feb0-4eac-96e9-ca7802d1f9d6.png)

**Comment by jonathanlermitage** [comment]
Sure, good idea! Some folders are already supported, mostly Liquibase, Flyway, some Python tools, and git sub-module folders.  
We can probably support more technologies. I think we can reuse the default folder icon (![folder](https://user-images.githubusercontent.com/857535/163517187-1d935f60-2e9c-4321-abdd-e9a625f7b541.svg)), change its color and a badge.  
Don't hesitate to do suggestions and create issues.

For now I am working on something else: the concept of icons groups. I would like to _tag_ a set of icons (it would be hardcoded) and let the user activate or deactivate this icons set. Per example, AngularJS icons would be tagged as "angular" icons, and the user would be able to activate or deactivate all AngularJS icons with a single click. For now, he has to select the icons individually, this is annoying ^^.  
It will take some time, but this is a background task. If I can create of integrate folder icons for popular technologies, I will do this with pleasure.

**Reply by franmc01** [reply]
friend just in case you have the folder in svg format?

**Reply by jonathanlermitage** [reply]
Sure! You can visit https://jetbrains.design/intellij/resources/icons_list/ and filter with "folder" :-)
